webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderUserDetailsFile.scala

```diff
@@ -146,16 +149,16 @@ object UserDetailList {
    *   - users with same username (according to the case sensitivity)
    *     will be removed from the returned list
    */
-  def sanityLoginFilter(userDetails: List[RudderUserDetail]): Map[String, RudderUserDetail] = {
+  def filterByCaseSensitivity(userDetails: List[RudderUserDetail], isCaseSensitive: Boolean): Map[String, RudderUserDetail] = {
```

Comment:
a function must never ever rely on a global configuration. That's a blinking red flag. We know it will cause stange errors, hard to refactor code, and other unsustainable problem. Dont' do that, ever. 

On some hypothesis, we can use global values at class level (ie directly referenced in class, not in there parameter). 
And there might be exception for default, "the world is set up like that" values, but the general rule of thumb is that we have bad intuition on them, so just add a parameter to the function and define it at class level.